news/2018-06-15-cwc-2018-recap-group-stage.md

```diff
@@ -0,0 +1,43 @@
+
+This match in particular will likely be remembered for a different reason however, in that, with the scoreline at 1-0 in Germany’s favour, Sweden elected to pick the perennially popular [Hige Driver join. SELEN - Dadadadadadadadadada](https://osu.ppy.sh/b/771881&m=2) from the NoMod pool. With both teams feeling red-hot off the back of previous victories, neither was prepared to give an inch to the other. The unthinkable happened, as all six players recorded an identical SS performance, resulting in a tie. But then the unthinkable happened again. And again. And even a fourth time – check the [match history](https://osu.ppy.sh/community/matches/43236583)! At the fifth time of asking, finally the German team made a mistake and allowed Sweden to tie the scores at 1-1, in what was effectively the 6th game of the match. Whilst Germany would eventually go on to take the match victory, both teams showed their incredible skill and determination here, and should be commended.
+
+## Upcoming Stage – Round of 16
+
+After 48 matches crammed with more tension and excitement than is probably safe for the human body to endure, we look forward to watching the 16 remaining teams do battle in the knockout stages from this coming weekend onward. On the top side of the bracket, Chile will open proceedings against Mexico, whilst on the other side of the globe, last year’s Champions South Korea look to build momentum against Japan. Later in the day, Norway and Finland clash with Hong Kong and Taiwan, respectively.
+
+Some truly terrifying monster matchups await us in the lower half of the bracket, however, with Poland once again taking on China, Germany crossing swords with the Netherlands, Sweden stepping up against the USA, and France taking the field against the 2016 runners-up, Indonesia. Who will rise and who will fall? Only time will tell...
+
+## Summary
+
+Whether you’re an avid supporter of your favourite team, or a casual fan hoping to see some exceptional feats of skill, endurance and strategy, this weekend’s Round of 16 matchups are absolutely not to be missed! Make sure to stay tuned to the [osu!live Twitch channel](https://www.twitch.tv/osulive) to catch all the action, and follow [@osugame on Twitter](https://twitter.com/osugame) to stay notified of updates, scheduling and live broadcasts.
+
+Additionally, if you want to show your undying love and support for your favourite teams and players, profile banners for the 16 knockout stage teams will be available to purchase from the [osu!store](https://osu.ppy.sh/store/listing) very soon! Keep your eyes peeled people!
+
+Hopefully you’re now all as excited for the remainder of this osu!catch World Cup as I am, and I look forward to seeing you all this weekend for more thrills, chills and inevitable fruit spills!
+
+**–JBHyperion**
```

Comment:
don't bold and use an actual em-dash for signoff

Reply:
Broken formatting when I copied from google docs or something I guess